frontend/src/components/placementVisuals/AddPlacementContainer.tsx

```diff
@@ -58,12 +59,29 @@ const AddPlacementContainer = ({ onConfirm, existingContainers }: AddPlacementCo
         if (container.samples.length > 0) {
             const loadedSamples: FMSSample[] = (await dispatch(api.samples.list({ id__in: container.samples.join(','), limit: 100000 }))).data.results
 
+            const projectIDs = new Set(loadedSamples.map(sample => sample.project))
+            const promises: Promise<any>[] = []
+            for (const projectID of projectIDs) {
+                if (projectID) {
+                    promises.push(dispatch(getProject(projectID)))
+                }
+            }
+            await Promise.all(promises)
+
+            const coordinates = selectCoordinatesByID(store.getState())
+            const projects = selectProjectsByID(store.getState())
+
             loadedSamples.forEach(sample => {
-                newDestination[sample.id] = { id: sample.id, coordinates: (sample.coordinate && coordinates[sample.coordinate].name) as string, name: sample.name }
+                newDestination[sample.id] = {
+                    id: sample.id,
+                    coordinates: sample.coordinate ? coordinates[sample.coordinate]?.name ?? '...' : 'N/A',
+                    name: sample.name,
+                    project: sample.project ? projects[sample.project]?.name ?? '...' : 'N/A'
```

Comment:
not a fan of N/A ... empty or undefined would be good... same goes for coordinates ...